go/genkit/schemas.config

```diff
@@ -144,16 +148,27 @@ ToolResponsePartToolResponse	pkg ai
 Part							pkg ai
 TextPart						pkg ai
 TextPart						name textPart
+TextPart.data					omit
```

Comment:
Any idea what these new `Part` fields are used for?  Should we be using them in document.go somehow?

Reply:
I've asked Keith to look into this. I just got things working so I could continue my API refactoring.

Reply:
I think the js side just split the part that we call "blob" into two, "media" and "data". I will make that same change in our code.